docs/int/quickstart/group/quickstart-group-cli.md

```diff
@@ -69,6 +68,8 @@ Every cluster member then participates in the DKG ceremony. For Charon v1, this
 docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.10.0 dkg --p2p-bootnode-relay
 ```
 
+>This is a helpful [video walkthrough](https://www.youtube.com/watch?v=94Pkovp5zoQ&ab_channel=ObolNetwork).
```

Comment:
Is this the newest video from yesterday @thomasheremans ?

Reply:
It is not. This is the old DKG recording we made which is still relevant for the CLI quickstart. I have added the new 1-min overview video at the top of the launchpad guide